Podfile.lock

```diff
@@ -61,4 +61,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: e5c47d206a7ea4550dad818207f3a4c8866cf9c0
 
-COCOAPODS: 1.9.3
+COCOAPODS: 1.9.2
```

Comment:
if 1.9.3 is already checked in, let's stick with that and upgrade ur local cocoapods version to 1.9.3 to avoid downgrades every time

Reply:
Changed